x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/action/PutDataFrameTransformAction.java

```diff
@@ -58,6 +58,11 @@ public Request(DataFrameTransformConfig config) {
         public Request(StreamInput in) throws IOException {
             super(in);
             this.config = new DataFrameTransformConfig(in);
+            if (in.getVersion().onOrAfter(Version.CURRENT)) {
+                this.deferValidation = in.readBoolean();
+            } else {
+                this.deferValidation = false;
+            }
         }
 
         public static Request fromXContent(final XContentParser parser, final String id) throws IOException {
```

Comment:
This method doesn't consider the new field "deferValidation". Is that ok?

Reply:
Only if I don't add the variable to the ctor. Right now the request assumes that a config is the payload and `deferValidation` is a REST param. I think that this particular input is just fine. I could add the variable as a parameter here and mark it final